server/game/cards/09.2-BoB/UtakuRumaru.js

```diff
@@ -0,0 +1,40 @@
+const DrawCard = require('../../drawcard.js');
+const { Players, CardTypes, Locations } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl');
+
+class UtakuRumaru extends DrawCard {
+    setupCardAbilities() {
+        this.persistentEffect({
+            match: (card) => card.isHonored && card.type === CardTypes.Character,
+            targetController: Players.Self,
+            effect: AbilityDsl.effects.modifyGlory(1)
+        });
+
+        this.persistentEffect({
+            match: (card) => card.isDishonored && card.type === CardTypes.Character,
+            targetController: Players.Self,
+            effect: AbilityDsl.effects.modifyGlory(-1)
+        });
+
+        this.reaction({
+            title: 'honor a participating character',
+            when: {
+                afterConflict: (event, context) => context.source.isParticipating() && event.conflict.winner === context.player
+            },
+            cost: AbilityDsl.costs.discardCard({
+                location: Locations.Hand,
+                targets: true
+            }),
+            target: {
+                cardType: CardTypes.Character,
+                controller: Players.Self,
```

Comment:
It can actually honor any participating character, so this should be `controller: Players.Any`